src/content/2.2/code/agda/snippet17.agda

```diff
@@ -0,0 +1,3 @@
+record ToString (a : Set) : Set where
+  constructor toString
+  field runToString : a → String
```

Comment:
Haskell version provide defintion of ToString and instance of contravariant functor for ToString.

Here we only have definition of ToString, so this snppet should have also following instance definition from the `Section212.agda`:
```Agda
  instance
    tostring : Contravariant ToString
    tostring .contramap f (toString g) = toString (g ∘ f)
 ```